utb/tests/test_add_user.py

```diff
@@ -1,20 +1,32 @@
-from django.http import HttpResponseBadRequest, HttpResponse, HttpRequest
+from django.http import HttpResponseBadRequest, HttpResponse, HttpRequest, HttpResponseForbidden
 from django.test import TestCase
-from .views import add_user
+from unittest.mock import Mock
 
+from utb import utils
+from utb.views import add_user
+from utb.models import User
+
+
+class AddUserTestCreation(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.cls_atomics = cls._enter_atomics()
+        # mocks the google_token check
+        utils.check_google_token = Mock(return_value=(True, None))
 
-class ViewsTestCase(TestCase):
     def test_null_or_empty_uid(self):
         uid = None
         name = "name"
         surname = "surname"
         email = "email@email.com"
         expected = HttpResponseBadRequest("Invalid arguments")
+
         response = add_user(HttpRequest(), uid, name, surname, email)
-        self.assertEqual(response.content, expected.content)
+        self.assertEquals(str(response), str(expected))
```

Comment:
is this str() necessary

Reply:
yes because if not the equal fail even if the HttpResponse is exactly the same. I tried to find a way without the str but basically what the str does is printing every field of the response so it is basically the behavior that I want